crates/rnote-ui/src/appwindow/appsettings.rs

```diff
@@ -446,9 +443,14 @@ impl RnAppWindow {
 
         {
             // Appwindow
-            app_settings.set_int("window-width", self.width())?;
-            app_settings.set_int("window-height", self.height())?;
-            app_settings.set_boolean("is-maximized", self.is_maximized())?;
+            if self.is_maximized() {
```

Comment:
I'd simplify this logic by only saving the width, height when it is !maximized, and unconditionally saving maximized